src/plugins/python/pythonbind.cpp

```diff
@@ -10,7 +10,7 @@
 
 
 #if PY_VERSION_HEX >= 0x03000000
-#if PY_VERSION_HEX >= 0x03050000
+#if PY_VERSION_HEX >= 0x03050000 && 0   // <- TODO: this is not Py_LIMITED_API compatible?
```

Comment:
`PyAsyncMethods` is not available in the limited API, so that check should probably be:
```c++
#if PY_VERSION_HEX >= 0x03050000 && !defined(Py_LIMITED_API)
```